vuu-ui/packages/vuu-filters/src/__tests__/__component__/FilterClause.cy.tsx

```diff
@@ -49,4 +50,20 @@ describe("FilterClause", () => {
       cy.findAllByRole("option", { name: "GBP" }).should("be.visible");
     });
   });
+
```

Comment:
would be good to add a test to demonstrate the `touched` so focus into the control, assert that no dropdown renders backspace to remove operator, select an operator, then show that dropdown now appears autotmatically